sdk-internals/communicator/ssh/connect.go

```diff
@@ -62,6 +63,8 @@ func BastionConnectFunc(
 			return nil, fmt.Errorf("Error connecting to bastion: %s", err)
 		}
 
+		log.Printf("[DEBUG] connected to bastion, hopping to destination")
```

Comment:
nit: hopping might not be clear for our non-native English speakers. 
```suggestion
		log.Println("[DEBUG] connected to bastion host")
		log.Println("[DEBUG] attempting connection to destination host")
```

Reply:
Rerolled with this change, thanks!